packages/subproviders/src/subproviders/ledger.ts

```diff
@@ -223,4 +196,39 @@ export class LedgerSubprovider extends BaseWalletSubprovider {
         this._ledgerClientIfExists = undefined;
         this._connectionLock.release();
     }
+    private async _initialDerivedKeyAsync(): Promise<DerivedHDKey> {
+        const ledgerClient = await this._createLedgerClientAsync();
+
+        let ledgerResponse;
+        try {
+            ledgerResponse = await ledgerClient.getAddress(
+                this._derivationBasePath,
+                this._shouldAlwaysAskForConfirmation,
+                SHOULD_GET_CHAIN_CODE,
+            );
+        } finally {
+            await this._destroyLedgerClientAsync();
+        }
+        const hdKey = new HDNode();
+        hdKey.publicKey = new Buffer(ledgerResponse.publicKey, 'hex');
+        hdKey.chainCode = new Buffer(ledgerResponse.chainCode, 'hex');
+        return {
+            hdKey,
+            address: ledgerResponse.address,
+            isChildKey: true,
+            derivationBasePath: this._derivationBasePath,
+            derivationPath: `${this._derivationBasePath}/0`,
+            derivationIndex: 0,
```

Comment:
Reference a single constant set to `0` in both the `derivationPath` and `derivationIndex`.